packages/monitors/test/CRMonitor.js

```diff
@@ -242,16 +242,28 @@ contract("CRMonitor.js", function(accounts) {
         // their position at a collateralization ratio of 175/(100*1)=1.75. monitoredSponsor is at 300/(100*1)=3.00.
         await emp.requestWithdrawal({ rawValue: convert("75") }, { from: monitoredTrader });
 
+        // The wallet CR should reflect the requested withdrawal amount.
```

Comment:
This is the unit test that we needed to test requested withdrawals factored into CR